src/metrics/codesmells_metrics.nit

```diff
@@ -0,0 +1,268 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+module codesmells_metrics
+
+# We usualy need specific phases
+# NOTE: `frontend` is sufficent in most case (it is often too much)
+import frontend
+
+import metrics_base
+import mclasses_metrics
+import semantize
+
+import method_analyse_metrics
+
+class BadConceptions
+	#Code smell list
+	var badConceptionElement = new Array[BadConception]
```

Comment:
Generally when declaring attributes or variables, the preferred Nit style is snake_case instead of camelCase